Automattic/newspack-plugin
# Testing with testing library

### All Submissions:






### Changes proposed in this Pull Request:

1. Fixes testing by adding missing preset to babel config
1. Changes the testing library from `enzyme` to… [`testing-library`](https://testing-library.com/)

Here's [a nice article](https://kentcdodds.com/blog/why-i-never-use-shallow-rendering) about how `enzyme` tests are unreliable and harder to maintain.

### How to test the changes in this Pull Request:

1. run `$ npm i` to install dependencies 
2. run `$ npm test` and observe tests passing
3. inspect test files and marvel at the expressivity of `testing-library` (vs. `enzyme`)

### Other information:
